src/content/configuration/optimization.md

```diff
@@ -200,6 +200,32 @@ module.exports = {
 };
 ```
 
+## `optimization.chunkIds`
+
+`bool: false` `string: natural, named, size, total-size`
```

Comment:
Don't get this notation, it allows `boolean` and `string`?
What do you think about putting it in a new line?